src/lib/supercomputer.ts

```diff
@@ -98,52 +98,37 @@ export class Supercomputer {
   }
 
   /**
-   * If child is top level, then initializes `this.olderChild`.
-   * If `children` object contains `children` property, it means inputs are nested (calls compute again).
+   * If `children` object contains `children` property, it means inputs are nested (calls computeChildren again).
    * Otherwise enriches inputs, passes them to Observatory.
    * For each model from pipeline Observatory gathers inputs. Then results are stored.
    */
   private async calculateOutputsForChild(
     childrenObject: Children,
-    params: any
+    childName: string
```

Comment:
Since we dropped `areChildrenNested`, the second object param was replaced with a string `childName`